test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -885,13 +885,34 @@ await ApiHubTestHelper.AssertTextUpdatedAsync(
         
 #endif
 
-        public class TestFixture : EndToEndTestFixture
+        public class TestFixture(IMessageSink sink)
+            : EndToEndTestFixture(rootPath, "node", RpcWorkerConstants.NodeLanguageWorkerName)
         {
             private static readonly string rootPath = Path.Combine("TestScripts", "Node");
+            private readonly AzuriteFixture _azurite = new(sink);
+
+            public override async Task InitializeAsync()
+            {
+                await _azurite.InitializeAsync();
+                await base.InitializeAsync();
+            }
 
-            public TestFixture()
-                : base(rootPath, "node", RpcWorkerConstants.NodeLanguageWorkerName)
+            public override async Task DisposeAsync()
             {
+                await base.DisposeAsync();
+                await _azurite.DisposeAsync();
+            }
+
+            public override void ConfigureScriptHost(IConfigurationBuilder configBuilder)
+            {
+                string connectionString = _azurite.GetConnectionString();
+                configBuilder.AddInMemoryCollection(new Dictionary<string, string>
+                {
+                    { "AzureWebJobsStorage", connectionString },
+                    { "ConnectionStrings:AzureWebJobsStorage", connectionString },
```

Comment:
Curious why we need to set AzureWebJobsStorage twice in two different ways?

Reply:
Shotgun approach. We support both, some tests use one or the other value.